sdk/synapse/synapse-artifacts/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0-beta.9 (2022-03-05)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+Added more supported types in `LinkedServiceOperations`
 
 ### Other Changes
 
+Upgrade `ArtifactsClient` to CoreV2 client
```

Comment:
I'd remove this comment. Maybe change it for `Taking latest codegen changes which include merging Client and ClientContext classes`

Reply:
Thanks for you comments!